### PR TITLE
HOT-2039 Address is not showing correctly, changed zip to 9 digits in…

### PR DESCRIPTION
**Pull request by mariamcws**
…stead of 10

### Jira Story

- [Address is not showing correctly HOT-2039](https://osi-cwds.atlassian.net/browse/HOT-2039)

## Description

Address was not showing correctly, changed zip to 9 digits instead of 10 that was previously permitted
## Tests
- [ ] I have included unit tests 
- [x] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by mariamcws** [comment]
@wmitchell-cambria  Yes, I am waiting for confirmation on acceptance criteria before merging.

File: spec/features/screening/incident_information_spec.rb [hunk 1]
```diff
@@ -53,8 +53,8 @@
 
   scenario 'character limitations by field' do
     within '#incident-information-card' do
-      fill_in 'Zip', with: '9i5%6Y1 8-_3.6+9*7='
-      expect(page).to have_field('Zip', with: '95618-3697')
+      fill_in 'Zip', with: '9i5%6Y1 8_3.6+9*7='
+      expect(page).to have_field('Zip', with: '956183697')
```
Comment (money) [inline spec/features/screening/incident_information_spec.rb]: According to the reported bug "_Structure of the desired result is xxxxx-xxxx_".  This test shows it breaks that expectation.

File: app/javascript/views/IncidentInformationForm.jsx [hunk 2]
```diff
@@ -74,7 +74,7 @@ const IncidentInformationForm = ({incidentDate, errors, onChange, onBlur, addres
           gridClassName='col-md-4'
           id='zip'
           label='Zip'
-          maxLength='10'
+          maxLength='9'
```
Comment (money) [inline app/javascript/views/IncidentInformationForm.jsx]: It should be 10 characters because of the hyphen.

**Comment by money** [comment]
oops, i see will already commented on the same things i mentioned.

File: spec/features/screening/incident_information_spec.rb (hunk 1 above)
Comment (mariamcws) [reply spec/features/screening/incident_information_spec.rb]: yes, the description of the story is conflicting. @JeffDent is going to review and refine the acceptance criteria.

**Comment by mariamcws** [comment]
@wmitchell-cambria  Made some changes based on discussion with @crystalctaylor. Now applying a mask to incident address zip in the format xxxxx-xxxx . Please review.

**Comment by mariamcws** [comment]
@wmitchell-cambria good catch, let me look into it some more

**Comment by mariamcws** [comment]
@wmitchell-cambria  As discussed in scrum, changing the Incident Address Zip Code to accept 5 digits.

**Pull request merged**